test/api/handlers/projects/create.js

```diff
@@ -27,6 +27,31 @@ after(function(done) {
   server.stop(done);
 });
 
+function cleanup(id, callback) {
+  server.inject({
+    url: '/projects/' + id,
+    method: 'delete',
+    headers: {
+      authorization: 'token ag-dubs'
```

Comment:
can we change this to a `test` token, so we're not baking people's usernames into our tests?